src/main/java/seedu/thanepark/logic/commands/UpdateCommand.java

```diff
@@ -223,6 +231,7 @@ public boolean equals(Object other) {
                     && getMaintenance().equals(e.getMaintenance())
                     && getWaitTime().equals(e.getWaitTime())
                     && getZone().equals(e.getZone())
+                    && getStatus().equals(e.getStatus())
```

Comment:
Hey Engsea, I think the naming for `UpdateRideDescriptor e` can be improved. For your consideration.